wiki/Gameplay/Grade/en.md

```diff
@@ -17,39 +26,33 @@ needs_cleanup: true
 | C | Over 60% 300s |
 | D | Anything else |
 
-A silver S or SS can be obtained by using the *[Hidden](/wiki/Game_modifier/Hidden)* and/or *[Flashlight](/wiki/Game_modifier/Flashlight)* mod.
-
 ## osu!taiko
 
 | Grade | Condition |
 | :-: | :-- |
-| SS | 100% Accuracy. |
-| S | Over 95% Accuracy (90+% GREATs, or less than 1 GOOD for every 10 notes). |
-| A | Over 90% Accuracy (80+% GREATs, or less than 1 GOOD for every 5 notes). |
-| B | Over 80% Accuracy (70+% GREATs, or less than 1 GOOD for every 3.33 notes). |
+| SS | 100% accuracy |
+| S | Over 95% accuracy (90+% GREATs, or less than 1 GOOD for every 10 notes) |
+| A | Over 90% accuracy (80+% GREATs, or less than 1 GOOD for every 5 notes) |
+| B | Over 80% accuracy (70+% GREATs, or less than 1 GOOD for every 3.33 notes) |
```

Comment:
This table should say:

```suggestion
| SS | 100% accuracy |
| S | Over 90% GREATs and no misses |
| A | Over 80% GREATs and no misses **OR** over 90% GREATs |
| B | Over 70% GREATs and no misses **OR** over 80% GREATs |
| C | Over 60% GREATs |
| D | Any other passing score |
```

So the grade requirements are actually closer to osu!'s than the previous info here would indicate. I've verified the above empirically on a test map with 20 hits.